sql/src/test/java/org/apache/druid/sql/calcite/CalciteQueryTest.java

```diff
@@ -12319,61 +12313,64 @@ public void testRequireTimeConditionLogicalValuePositive()
   @Test
   public void testRequireTimeConditionSimpleQueryNegative()
   {
-    msqIncompatible();
-    expectedException.expect(CannotBuildQueryException.class);
-    expectedException.expectMessage("__time column");
+    Throwable exception = assertThrows(CannotBuildQueryException.class, () -> {
+      msqIncompatible();
```

Comment:
nit: Consider moving `msqIncompatible()` and any other test initialization outside `assertThrows`.
Same applies to the other tests